src/component/Restaurants.js

```diff
@@ -63,9 +93,45 @@ function Restaurants (props) {
                     </select>
                 </div>
             </form>
-
+            {restaurantList}
         </div>
     );
 }
 
+Restaurants.propTypes = {
+    selected: PropTypes.func.isRequired
+};
+
 export { Restaurants };
+
+const DEFAULT_SELECT_VALUE = "";
+const IMAGE_SRC_MAP = Object.freeze({
+    "node_modules/place-my-order-assets/images/1-thumbnail.jpg": thumbnail1Src,
```

Comment:
I wasn't in the conversation, but I thought the API was adjusted so that it would serve them directly?

Reply:
Yes

Reply:
This has been removed.